app/uk/gov/hmrc/tradergoodsprofilesrouter/models/CreateRecordPayload.scala

```diff
@@ -30,17 +30,23 @@ case class CreateRecordPayload(
   goodsDescription: String,
   countryOfOrigin: String,
   category: Int,
-  assessments: Option[Seq[Assessment]],
-  supplementaryUnit: Option[Int],
-  measurementUnit: Option[String],
+  assessments: Option[Seq[Assessment]] = None,
+  supplementaryUnit: Option[Int] = None,
+  measurementUnit: Option[String] = None,
   comcodeEffectiveFromDate: Instant,
-  comcodeEffectiveToDate: Option[Instant]
+  comcodeEffectiveToDate: Option[Instant] = None
 )
 
 object CreateRecordPayload {
   implicit val format: OFormat[CreateRecordPayload] = Json.format[CreateRecordPayload]
 
-  def apply(eori: String, incomingRequest: CreateRecordRequest): CreateRecordPayload =
+  def apply(eori: String, incomingRequest: CreateRecordRequest): CreateRecordPayload = {
```

Comment:
do we really need this one. Assesment call the removeNull on the json write which remove the null value already